src/gt4py/next/iterator/transforms/pass_manager.py

```diff
@@ -137,29 +138,30 @@ def apply_common_transforms(
     if lift_mode != LiftMode.FORCE_INLINE:
         # FIXME[#1582](tehrengruber): implement new temporary pass here
         raise NotImplementedError()
-        assert offset_provider is not None
-        ir = CreateGlobalTmps().visit(
-            ir,
-            offset_provider=offset_provider,
-            extraction_heuristics=temporary_extraction_heuristics,
-            symbolic_sizes=symbolic_domain_sizes,
-        )
-
-        for _ in range(10):
-            inlined = InlineLifts().visit(ir)
-            inlined = InlineLambdas.apply(
-                inlined, opcount_preserving=True, force_inline_lift_args=True
-            )
-            if inlined == ir:
-                break
-            ir = inlined
-        else:
-            raise RuntimeError("Inlining 'lift' and 'lambdas' did not converge.")
-
-        # If after creating temporaries, the scan is not at the top, we inline.
-        # The following example doesn't have a lift around the shift, i.e. temporary pass will not extract it.
-        # λ(inp) → scan(λ(state, k, kp) → state + ·k + ·kp, True, 0.0)(inp, ⟪Koffₒ, 1ₒ⟫(inp))`
-        ir = _inline_into_scan(ir)
+        # ruff: noqa: ERA001
```

Comment:
Why is this commented out and not deleted?

Reply:
Because the implementation is still missing and this is the template for it. It will be deleted in the next PR. Not sure why I commented it out, but doesn't matter.